### DISCUSSION TITLE
Multi-Tenancy Support

### DISCUSSION THREAD

**Discussion by vgiattino** [open-metadata/OpenMetadata#8224] [General]
Does Open Metadata support the concept of Multi-Tenancy?

**Comment by harshach** [comment]
@vgiattino we have roles and policies that will restrict access to the metadata based on the policy definition. Let us know if thats what you are looking for. Also please reach us at https://slack.open-metadata.org we are more active there

**Reply by vgiattino** [reply]
Looking to run a single instance of the software and serve multiple customers/departments as tenants with data isolated to that tenant.

**Reply by ayush-shah** [reply]
Hey @vgiattino, Yes, we support that as well.
